src/elements/content-sidebar/activity-feed/task-form/TaskForm.js

```diff
@@ -357,7 +384,7 @@ class TaskForm extends React.Component<Props, State> {
                         <PillSelectorDropdown
                             className={pillSelectorOverlayClasses}
                             error={this.getErrorByFieldname('taskAssignees')}
-                            disabled={isLoading}
+                            disabled={isLoading || isForbiddenErrorOnEdit}
```

Comment:
This logic is super repeated. Should be made a new const above, isDisabled= and used wherever applicable

Reply:
Personally, i find it easier to reason about why it's disabled when the evaluation has parts like this, but to your point, there are several that need to match, and duplicating these two is a potential source of future bugs